meta-xt-control-domain/recipes-guest/doma/doma.bb

```diff
@@ -10,6 +10,8 @@ inherit externalsrc systemd
 # We use custom U-BOOT to run the Android
 RDEPENDS_${PN} = "u-boot-android"
 
+XT_DOMA_CONFIG_NAME ?= "doma-generic.cfg"
```

Comment:
The whole idea is to force user configure config name via YAML file. Because, AFAIK there is no truly "generic" config that can work on any platform.

So I'd prefer to drop this patch.

Reply:
Skipped